src/coreclr/jit/valuenum.cpp

```diff
@@ -2845,7 +2849,11 @@ ValueNum ValueNumStore::EvalFuncForConstantArgs(var_types typ, VNFunc func, Valu
             }
             else
             {
+#ifdef TARGET_LOONGARCH64
+                assert(typ == TYP_INT || typ == TYP_LONG);
+#else
```

Comment:
Please add a comment describing why this is platform specific.

Reply:
OK, I will add a comment.